src/common/options/rgw.yaml.in

```diff
@@ -1477,6 +1480,7 @@ options:
   desc: File-system path for ops log.
   long_desc: Path to file that RGW will log ops logs to.
   fmt_desc: The file-system path for writing operations logs.
+  daemon_default: /var/log/ceph/ops-log-$cluster-$name.log
```

Comment:
note that i copied this from the `log_file` option which defaults to `/var/log/ceph/$cluster-$name.log`

Reply:
@cfsnyder i'd like to add a blurb about log rotation for `rgw_ops_log_file_path`. can you summarize how cephadm does this, or point me at docs maybe?

Reply:
Yeah sure @cbodley. cephadm automatically adds logrotate config on the host for all files matching `/var/log/ceph/{fsid}/*.log`. So if the `rgw_ops_log_file_path` config param matches `/var/log/ceph/*.log` (container path), it'll be handled by the cephadm config.

Here's an example of the config it generated on one of my test clusters:

```
root@ceph01:~# cat /etc/logrotate.d/ceph-74016538-b125-11ec-abb9-3daf21fa986b 
# created by cephadm
/var/log/ceph/74016538-b125-11ec-abb9-3daf21fa986b/*.log {
    rotate 7
    daily
    compress
    sharedscripts
    postrotate
        chown 167:167 /var/log/ceph/74016538-b125-11ec-abb9-3daf21fa986b/*.log; killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x 'ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror' || true
    endscript
    missingok
    notifempty
    su root root
}
```

As you can see, it rotates daily and keeps 7 days of compressed logs.

Reply:
that's great, thanks